e2e/testdata/fn-render/fn-failure-output-no-truncate/.expected/config.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# One of the functions in the pipeline fails resulting in
+# non-zero exit code and no changes in the resources.
+exitCode: 1
+disableOutputTruncate: true
+stdErr: "ERR  - stdin: Failed initializing schema file:///tmp/master-standalone-strict/custom-custom-v1.json: open /tmp/master-standalone-strict/custom-custom-v1.json: no such file or directory"
```

Comment:
didn't @mengqiy update this already?

Reply:
The real error message is `Invalid type. Expected: [integer,null], given: string in object 'apps/v1/Deployment//nginx-deployment' in file resources.yaml in field spec.replicas`. Will update this test case.

Reply:
This is WAI, because this is checking an arbitrary CRD and the kubeval doesn't have schema.
The behavior of kubeval is that it tries to look it up from the schema location and error out if it can't find it. Either `ignore_missing_schemas` or `skip_kinds` can be used to avoid this error.

The error message can be improved to suggest user to use `ignore_missing_schemas` or `skip_kinds`.